libs/platform/table/components/table-view-settings-dialog/grouping/grouping.component.html

```diff
@@ -1,9 +1,9 @@
+<h3 fd-list-group-header>
+    <span fd-list-title [attr.id]="groupOrderHeaderId">
+        {{ 'platformTable.groupDialogGroupOrderHeader' | fdTranslate }}
+    </span>
+</h3>
```

Comment:
the heading level should be customizable. We can't know for sure the sequence of elements on the apps so hardcoding `h3` could cause problems. We can have it as default, but should add an option to modify it. I suggest you use a div with role heading and have the heading level be set by an input prop.